### DISCUSSION TITLE
Status_code': 401, 'error': 'Unauthorized', 'message': 'User not found', 'extensions': {'code': 'AUTH_ERROR'}}

### DISCUSSION THREAD

**Discussion by Sahuism** [IBM/ibm-generative-ai#104] [Q&A]
Hello

I am getting following unauthorized access error while running Async and Synchronus examples from Getting Started page:
DEBUG:genai.services.async_generator:Received response = {'status_code': 401, 'error': 'Unauthorized', 'message': 'User not found', 'extensions': {'code': 'AUTH_ERROR'}}

I have already created .env file and also added SSL .py file. Please anyone suggest how to fix this error.
When enabled logs I got this message in the logs.

I am using IBM-GenAI Pynthon SDK on Windows.

**Comment by jayolee** [comment]
Hi Prashant, please check your API endpoint. 
Please reach out to us via internal channels for more help. 
If you have issues finding ways to internally contact us, please reach out to younglee@ibm.com Thanks.

**Reply by Sahuism** [reply]
Sure! Thanks for your reply.
